.changeset/blue-books-smash.md

```diff
@@ -0,0 +1,5 @@
+---
+"@sketch-hq/sketch-file": patch
```

Comment:
Curious to see how this is going to be versioned 😅 but I think with an initial release this could be almost anything `major`, `minor` or `patch`. As far as `sketch-file` is concerned it's "major" but the others are obviously unaffected.

In any case , changesets will preview the versioning later on. So all good 👍

Reply:
You’re right, looks like Changesets uses `patch` by default and that’s clearly not the case here. Will change.